build/yamls/flow-visibility/flow-visibility.yaml

```diff
@@ -139,7 +126,7 @@ spec:
           imagePullPolicy: IfNotPresent
           env:
             - name: GF_INSTALL_PLUGINS
-              value: "https://github.com/antrea-io/antrea/releases/download/v1.5.0/grafana-sankey-plugin;antreaflowvisibility-grafana-sankey-plugin,vertamedia-clickhouse-datasource"
+              value: "https://github.com/zyiou/grafana-custom-plugin/releases/download/1.0.6/grafana-sankey-plugin.zip;antreaflowvisibility-grafana-sankey-plugin,vertamedia-clickhouse-datasource"
```

Comment:
We'd better to change it back to the antrea/releases url before getting merged. 
Btw, since the PR will not be included in release v1.5.0, shall we change it to v1.6.0?

Reply:
Thanks, changed it to v1.6.0 with antrea/release url.